src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -250,6 +252,57 @@ private function fixFunctionDocComment($content, Tokens $tokens, $functionIndex,
         return $docBlock->getContent();
     }
 
+    /**
+     * @param string $content
+     * @param int    $docCommentIndex
+     *
+     * @return string
+     */
+    private function fixPropertyDocComment($content, Tokens $tokens, $docCommentIndex, array $shortNames)
+    {
+        $docBlock = new DocBlock($content);
+
+        $index = $tokens->getNextMeaningfulToken($docCommentIndex);
+
+        $kindsBeforeProperty = [T_STATIC, T_PRIVATE, T_PROTECTED, T_PUBLIC];
+
+        if (!$tokens[$index]->isGivenKind($kindsBeforeProperty)) {
+            return null;
```

Comment:
This violates the return type (`string`). Maybe this `if` should be done in the caller function so we never get to this point?

Reply:
Fixed.